src/Http/Resources/V1/Admin/Catalog/CategoryResource.php

```diff
@@ -18,13 +18,16 @@ public function toArray($request)
         return [
             'id'                 => $this->id,
             'name'               => $this->name,
+            'parent_id'          => $this->parent_id,
             'slug'               => $this->slug,
+            'position'           => $this->position,
             'display_mode'       => $this->display_mode,
             'description'        => $this->description,
             'meta_title'         => $this->meta_title,
             'meta_description'   => $this->meta_description,
             'meta_keywords'      => $this->meta_keywords,
-            'status'             => $this->status,
+            'visible_in_menu'    => $this->status,
```

Comment:
Change this to `status` as we can't rename the `status` key because some other users may use this key.

Reply:
fix